core/primitives/src/trie_key.rs

```diff
@@ -405,6 +405,19 @@ pub mod trie_key_parsers {
         res
     }
 
+    pub fn get_raw_prefix_for_access_key(
```

Comment:
You don't need this method. You can call the TrieKey.to_vec() instead. 

(the 'get_raw_prefix_for_access_keys' above was needed, as it was looking at a root for multiple keys - here you want to get the path to a concrete trie key)

Reply:
[e2cded2](https://github.com/near/nearcore/pull/8644/commits/e2cded2a2c205f5cf907dd1e22d0ca2833a6a63f)